releases/release-1.25/release-team.md

```diff
@@ -0,0 +1,17 @@
+# Kubernetes 1.25 Release Team
```

Comment:
On numeral occasions, I'm looking for the team members table and I can't find it easily, just to realize that is in another file `release-team.md` and on the README.md.

Would be too much trouble to put the able at the end of the README.md instead of a separate file?

Reply:
We have `links` at the top of readme file pointing to release team table: https://github.com/kubernetes/sig-release/blob/7d494b9a3bc717e6089b5ed5780b822c85abc841/releases/release-1.25/README.md?plain=1#L16

I hope that works^